R/readutils.R

```diff
@@ -343,7 +343,40 @@ readtextcf <- function(file, T=48, sym=TRUE, path="", skip=1, check.t=0, ind.vec
 
   return (invisible(ret))
 }
+#' @title reading reweighting factors for a list of gauge configuration 
+#'        and random samples from ASCII files in CMI format
```

Comment:
reweighting factors are not in CMI format

Reply:
still there...

Reply:
I do not see this now

Reply:
what do you mean you don't see it? the function is called `readcmi.rw` and the documentation says

```
#' @title reading reweighting factors for a list of gauge configuration 
#'        and random samples from ASCII files in CMI format 
```